src/main/java/com/aphyr/riemann/client/Promise.java

```diff
@@ -75,6 +99,11 @@ public Object unsafeDeref(final long time,
     }
   }
 
+  @Override
+  public void setCallback(Callback<T> c) {
```

Comment:
Naw, we're only on Netty 3 cuz I haven't had a chance to upgrade yet. Netty 4 should happen next. :)